main.py

```diff
@@ -75,6 +75,20 @@ def main():
         help="Browser to use for cookies.",
     )
 
+    parser.add_argument(
```

Comment:
What if both the browser and cookie file are specified?

Reply:
It tries to open it in browser and get failed

Reply:
Who is it? What does it open? And what gets failed?

Reply:
Who: Ytdlp
How: see the crashlog reported

Reply:
The crashlog was not caused by the cookie file as no cookie file was provided. My question is 

> What if both the browser and cookie file are specified?

Reply:
When we giving cookiesfrombrowser arg ytdlp tries to handle it no matter cookiefile is supplied or not. And in termux the cookiesfrombrowser doesn't get things and crashed.